### DISCUSSION TITLE
Can I show 3D models in search results, but not on the source website?

### DISCUSSION THREAD

**Discussion by andrewtsymbal** [google/model-viewer#1887] [Q&A]
### Description

How is it possible to embed 3d model so it will be visible only in Google AR Search but not visible on the page?
Below are 2 examples from Burberry and Wikipedia, their 3D models appear in search but not visible on the appropriate pages.

#### Live Demo
Burberry example:
https://www.dropbox.com/s/wpczqdthdugwu60/Google_AR_3D_Capture%20.mp4

Wikipedia example:
https://i.imgur.com/2eGW7HP.png

**Comment by cdata** [comment]
This is not currently supported, but I would first refer you to this comment [here](https://github.com/google/model-viewer/issues/1131#issuecomment-610900399)...

...and also point out that Google Search guidelines regarding structured data state explicitly:

> Don't mark up content that is not visible to readers of the page. For example, if the JSON-LD markup describes a performer, the HTML body should describe that same performer. ([source](https://developers.google.com/search/docs/guides/sd-policies))

> structured data on the page should describe the content of that page. You should not create blank or empty pages just to hold structured data; nor should you add structured data about information that is not visible to the user, even if the information is accurate. ([source](https://developers.google.com/search/docs/guides/intro-structured-data))

**Comment by andrewtsymbal** [comment]
Okay and what about showing 3d content only on mobile but invisible on PC version?

**Comment by ronpadz** [comment]
Allow me to share some info about the appearance of the "View in 3D" chip in search results that was shared with me by resources on the business development side of the Google Daydream team recently:
- This feature is in limited beta with a small handful of mostly large companies 
- They are being very selective about who can join the beta and will likely only consider organizations that have a very large number of models integrated into their site (i.e. >1000)
- This beta test is being used to work out the value proposition to the almighty Google search results scheme as well as the mechanics of how the facility should look and work
- There is no guaranty that the "View in 3D" search results chip will ever make it out of beta

**Reply by aviggiano** [reply]
@ronpadz We have a client that meets these guidelines (over 5000 3D/AR models on their website) but the [old Google form](https://developers.google.com/search/docs/guides/3d-ar) seems to be offline. Is there another way to apply for this beta?

**Reply by elalish** [reply]
We're working on opening this up right now, please stay tuned!

**Comment by cdata** [comment]
@andrewtsymbal there is obviously gray area here, because one cannot force a mobile experience to contain exactly the same content as the desktop experience.

That said, my two cents: I think it goes against the principle of publishing in general to "change the story" you tell to users based on the form factor of the medium. If I read a novel, it doesn't matter if I read it on paperback, hardcover or in an e-reader. Even if I watch a movie adaptation, my hope as a viewer is to be exposed to a specific story. Obviously some liberties need to be taken to adapt a written story for the big screen, but I'll tell you what: I was very disappointed that they dropped the giant squid from the Watchmen movie; it wasn't the story I was expecting.

If the content on the page relates the same details on PC and mobile, you are probably fine. But to the extent that a 3D model is an essential part of that content - and if you want it to be indexed by Search, I would say it rises to the level of an essential part of that content - it should be presented to all viewers of that content regardless of device type.
